php/docker/templates/nginx-vhost-conf.d/page-cache.d/10-index.conf

```diff
@@ -13,6 +14,15 @@
 
 set $skip_cache 0;
 
+set $key "";
+set $key_prefix {{ $keyPrefix }};
+set $key_uid {{ $keyUID }};
+set $key_uid_head {{ $keyUID | replace "$request_method" "GET" }};
```

Comment:
In my opinion, we can drop `$request_method` from the cache key.

Reply:
Yes, we could do that, but we'll have to make some changes in our nginx-helper fork, where the `GET` method is hard-coded in several places. I'll leave that for some other time, since there's no test written in there as well.
And srcache sort of offers support for caching `POST PUT and DELETE` methods if somebody desires so.

But ultimately I agree, and I will open another issue to track this cleanup.